SFBaseKit/Utils/Observable.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Observable.swift
+//  SFBaseKit
+//
+//  Created by Plamen Penchev on 19.05.21.
+//  Copyright © 2021 Upnetix. All rights reserved.
+//
+#if canImport(Combine)
+import Foundation
+import Combine
+
+@available(iOS 13, *)
+/// Object that represents an observable to which subscribers can attach.
+/// Current implementation relies entirely on Combines `CurrentValueSubject`,
+/// therefore it's available for iOS 13+ only.
+public final class Observable<T> {
+    private let subject: CurrentValueSubject<T, Never>
+    private var cancellables = Set<AnyCancellable>()
+    
+    /// The current value held in the observable.
+    public var value: T {
+        set {
```

Comment:
I suggest using willSet{} is better approach here because with set{} approach subject.send() cannot listen for changes because variable has already been set when using Observable in animations for example.